mangaki/mangaki/views.py

```diff
@@ -259,11 +259,10 @@ def post(self, request, *args, **kwargs):
 class WorkListMixin:
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
+        ratings = current_user_ratings(self.request, context['object_list'])
 
-        ratings = current_user_ratings(
-            self.request, list(context['object_list']))
         for work in context['object_list']:
-            work.rating = ratings.get(work.id, None)
+            work.rating = ratings.get(work.id)
```

Comment:
I wonder if there is a case where `ratings` could not contain the `work.id` we're considering?